ios/RNMagicScriptTests/sources/specs/UiNodes/Layouts/GridLayoutSpec.swift

```diff
@@ -296,6 +296,62 @@ class GridLayoutSpec: QuickSpec {
                     expect(layout.getSize()).to(beCloseTo(CGSize(width: 2 * referenceWidth, height: referenceHeight)))
                 }
             }
+
+            context("flow direction") {
+                it("should have vertical flow direction") {
```

Comment:
I think we should add additional context to describe why it should have vertical flow.